source/includes/_payment.md

```diff
@@ -3124,9 +2003,6 @@ class Payment {
    "auditLogs":[]
 }
 ```
```

Comment:
Same comment as above, I think an empty body is sent, could you please confirm and update if required?